doc/tutorials/java-kdb.md

```diff
@@ -11,30 +11,17 @@ In order to use `kdb` you need to include the dependency in your project. [Here]
 After that you can start loading a `KDB` object as follows:
 
 ```java
-var key = Key.createNameless();
-try (KDB kdb = KDB.open(key)) {
+try (KDB kdb = KDB.open()) {
     // code to manipulate keys
 } catch (KDB.KDBException e) {
     e.printStackTrace();
-} finally {
-    key.release(); // optional clean-up
+    e.releaseErrorKey(); // optional clean-up
```

Comment:
Why would this be necessary?

Reply:
When the cleaner is enabled and working, it is not. It's optional to manually release the native resource, if one wants to take control over when these resources are being released. The example code includes this for completeness.
Do we want to decide to leave out references to native resource release for all code examples?

Reply:
I think the tutorial should show the simplest way of doing things. You could add a hint that a manually releasing is an option. But unless you are very memory constrained, it's probably better to leave the cleanup to the garbage collector.

Reply:
I agree and have updated the doc accordingly.